### PR TITLE
Add recipe for psutil 5.4.7

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -100,7 +100,7 @@ before_script:
   - |
     for meta in "${WHEEL_METAS[@]}"; do
         _f=${meta#wheels/} ; wheel=${_f%%/*}
-        wheel_type=$($STARFORGE_CMD wheel_type --wheels-config="$meta" "$wheel")
+        wheel_type=$($STARFORGE_CMD wheel_type --wheels-config="$meta" "$wheel") || return $?
```
Comment (nsoranzo) [inline .travis.yml]: I don't understand why [this build](https://travis-ci.org/galaxyproject/starforge-recipes/jobs/424332534) is not failing here.

Comment (nsoranzo) [reply .travis.yml]: Swapping `return` with `exit` made the builds fail.

**Comment by natefoo** [comment]
meta.yml needs `type: wheel` for now.

**Comment by nsoranzo** [comment]
That was it, thanks @natefoo! Time to merge https://github.com/galaxyproject/starforge/pull/214 ?

**Comment by nsoranzo** [comment]
Having this on wheels.galaxyproject.org also fixed the building of the Galaxy docs on Jenkins, whoot!

**Pull request merged**